deploy-board/deploy_board/templates/deploys/edit_constraint_modal.tmpl

```diff
@@ -41,6 +41,7 @@
                                    data-placement="right"
                                    data-trigger="manual"
                                    title="numbers only." value="{{ max_parallel }}">
+                            <small>Note: This number MUST be smaller the config/max_parallel_hosts</small>
```

Comment:
smaller **than** ?

Reply:
thanks for finding this -- i updated the text and added a link to the env/stage config page as well for easy access to user

![image](https://user-images.githubusercontent.com/29519/76491011-d3ca4600-63e9-11ea-98c7-d01d9849f5e2.png)